common/lib/dependabot/shared_helpers.rb

```diff
@@ -82,6 +85,7 @@ def self.escape_command(command)
     end
 
     # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/AbcSize
```

Comment:
Looks like this disablement [isn't necessary](https://github.com/dependabot/dependabot-core/actions/runs/6710089882/job/18234551734?pr=8302#step:4:12) 😎